### ISSUE TITLE
[rock-dkms-firmware-bin] Outdated URL in PKGBUILD

### ISSUE THREAD

**Issue by rbberger** [rocm-arch/rocm-arch#299]
Trying to install the `rock-dkms-firmware` package fails with
```
curl: (22) The requested URL returned error: 404 Not Found
==> ERROR: Failure while downloading http://repo.radeon.com/rocm/apt/debian/pool/main/r/rock-dkms/rock-dkms-firmware_3.5-32_all.deb
```

### Reason:
```
source=("http://repo.radeon.com/rocm/apt/debian/pool/main/r/rock-dkms/rock-dkms-firmware_${_pkgver}_all.deb")
```
The file http://repo.radeon.com/rocm/apt/debian/pool/main/r/rock-dkms/rock-dkms-firmware_3.5-32_all.deb doesn't exist.

### Workaround:

Replace the source line in the PKGBUILD
```
source=("https://repo.radeon.com/rocm/apt/3.5.1/pool/main/r/rock-dkms/rock-dkms-firmware_${_pkgver}_all.deb")
```

**Comment by gmaleshkov** [comment]
Same issue as  #286

**Comment by acxz** [comment]
Thx for reporting, should be resolved with cae3851
